Source/MQTTnet/Implementations/PlatformAbstractionLayer.cs

```diff
@@ -1,20 +1,25 @@
-﻿using System.Threading.Tasks;
+﻿using System;
+using System.Threading.Tasks;
 
 namespace MQTTnet.Implementations
 {
     public static class PlatformAbstractionLayer
     {
-        public static Task CompletedTask
+        public static readonly byte[] EmptyByteArray = new byte[0];
+
+#if NET452
+        public static Task CompletedTask => Task.FromResult(0);
+#else
+        public static Task CompletedTask => Task.CompletedTask;
+#endif
+
+        public static void Sleep(TimeSpan timeout)
```

Comment:
this makes no sense to me just use Thread.Sleep if you need to blocking wait

Reply:
Thread.Sleep is not available for all build targets (when I am not wrong it is not available in UWP).